shell/src/main/java/alluxio/cli/UfsTests.java

```diff
@@ -0,0 +1,41 @@
+package alluxio.cli;
+
+import com.beust.jcommander.JCommander;
+
+public class UfsTests {
```

Comment:
This class is called by `runUfsTests`, enclosing all the command line logic from the old `UnderFileSystemContractTest` class